src/customRules/trailingCommaInterfaceRule.ts

```diff
@@ -1,6 +1,19 @@
 import * as ts from 'typescript';
 import * as Lint from 'tslint';
 
+class TrailingCommaInterfaceWalker extends Lint.RuleWalker {
```

Comment:
Convention in `tslint` is to put the walker after the rule (e.g., [unified-signatures-rule](https://github.com/palantir/tslint/blob/b4be183b8e3d9fa2e07d7e0ab34c500ee0b0bb7b/src/rules/unifiedSignaturesRule.ts#L59)).